### PR TITLE
Add social tab, buttons, cta

### PR DESCRIPTION
**Pull request by skmp**
Quick and dirty social tab, discord CTA in games list

Windows and android haven't been tested, hopefully they build.

@davidgfnet can we handle this in switch too?

reicast-cta-draft
reicast-social-draft

### CODE REVIEW

**Comment by gigaherz** [comment]
Hmmm, I feel like it would look better if that tab was called "Help" or such? I tend to actively ignore things called "social".

File: core/windows/winmain.cpp [hunk 1]
```diff
@@ -810,3 +812,9 @@ void os_DoEvents()
 
 int get_mic_data(u8* buffer) { return 0; }
 int push_vmu_screen(u8* buffer) { return 0; }
+
+void os_LaunchFromURL(const string& url)
+{
+	auto cmd = "start " + url; 
+	system(cmd.c_str());
```
Comment (gigaherz) [inline core/windows/winmain.cpp]: Ew. There's windows APIs for that, which don't involve the "start" program :P
e.g. 
```c
ShellExecuteA(0, 0, url.c_str(), 0, 0 , SW_SHOW );
```

Comment (skmp) [reply core/windows/winmain.cpp]: Mnn, can you implement it this way and append on the PR? I don't have a windows env setup, so this was 100% guesswork

Comment (gigaherz) [reply core/windows/winmain.cpp]: Done.

**Comment by skmp** [comment]
It is FB, discord, etc - that's why I named it social (for now). It's fine if people ignore it, for now, I guess :p

File: core/windows/winmain.cpp (hunk 1 above)
Comment (skmp) [reply core/windows/winmain.cpp]: Thx. Does it work as expected on windows?

Comment (gigaherz) [reply core/windows/winmain.cpp]: Yup! Executing the "open" verb of an URL will open it in the default browser, same as if you were opening a document.

**Pull request merged**